helpdesk/views/staff.py

```diff
@@ -457,7 +457,7 @@ def update_ticket(request, ticket_id, public=False):
     if request.FILES:
         import mimetypes, os
         for file in request.FILES.getlist('attachment'):
-            filename = file.name.encode('ascii', 'ignore')
+            filename = file.name
```

Comment:
It is always a good idea to check. Can not we solve this problem without django.utils.six :)? I think we can :) just check this code on py2.7 it should work even there.